index.js

```diff
@@ -40,40 +58,68 @@ document.addEventListener('DOMContentLoaded', () => {
     }
   }
 
-  const pressOperation = (el) => {
-    let localMemory = display.value
+  numberBtn.forEach((el) =>
+    el.addEventListener('click', (e) => {
+      pressNum(e.target.textContent)
+    })
+  )
 
-    if (isNewNumber && currentOperator !== '=') {
-      display.value = currValue
-    } else {
-      isNewNumber = true
-      if (currentOperator === '+') currValue += +localMemory
-      else if (currentOperator === '-') currValue -= +localMemory
-      else if (currentOperator === '*') currValue *= +localMemory
-      else if (currentOperator === '/') currValue /= +localMemory
-      else currValue = +localMemory
-      display.value = currValue
-      currentOperator = el
-    }
+  const KEYS = {
+    C: 'C',
+    CE: 'CE',
   }
 
   const pressClearBtn = (el) => {
-    if (el === 'C') {
+    if (el === KEYS.C) {
       display.value = 0
       currValue = 0
-    } else if (el === 'CE') {
+    } else if (el === KEYS.CE) {
       display.value = '0'
       isNewNumber = true
     }
   }
 
-  const pressOnPoint = (el) => {
-    console.log(el)
+  clearBtn.forEach((el) =>
+    el.addEventListener('click', (e) => {
+      pressClearBtn(e.target.textContent)
+    })
+  )
+
+  const pressOnPoint = () => {
     if (isNewNumber) {
       display.value = '0.'
       isNewNumber = false
-    } else if (el === 'point') {
+    } else {
       display.value += '.'
     }
   }
+
+  pointBtn.addEventListener('click', () => {
+    pressOnPoint()
+  })
+
+  document.addEventListener('keydown', (key) => {
+    console.log(key)
+    const maybeNumber = +key.key
+    const maybeOperator = key.key
```

Comment:
In this case I would not use two variables. You check maybeNumber once and then convert it back to string. Create only ONE variable here and convert it number where it is needed.